cardinal/types/message.go

```diff
@@ -5,6 +5,9 @@ type Message interface {
 	Name() string
 	// Group returns the group of the message.
 	Group() string
+	// FullName returns the fully qualified name of the message. It is formed by <group_name>.<message_name>.
+	// example: persona.create-persona
+	FullName() string
```

Comment:
I've added a triage ticket to collapse FullName and ID into the same thing: https://linear.app/arguslabs/issue/WORLD-980/in-cardinaltypesmessagego-id-and-fullname-should-be-combined

Reply:
thanks!